packages/backend/src/services/Territoire.js

```diff
@@ -45,9 +46,21 @@ const query = {
       FROM geo.territoires ter
       INNER JOIN back.fiche_territoire fte ON fte.ter_code = ter.code
       LEFT JOIN back.users usr ON usr.ter_code = ter.code
-      WHERE code <> 'FRA'
+      WHERE ter.code <> 'FRA'
+      ${search.map((s) => ` AND ${s} `).join("")}
```

Comment:
```suggestion
      ${search.map((s) => `AND ${s}`).join(" ")}
```

(je trouve ca un peu plus logique :D)